lib/Cake/basics.php

```diff
@@ -227,6 +226,7 @@ function h($text, $double = true, $charset = null) {
 		}
 		if (is_string($double)) {
 			$charset = $double;
+			$double = true;
```

Comment:
this is not similar to `master` version 
https://github.com/cakephp/cakephp/blob/21c8fce5ca8b68292b948a28606006faa935b579/src/Core/functions.php#L66-L71

Reply:
It is not clear what you mean by that. The fourth parameter to `htmlspecialchars` below must be a boolean. So, `$double` should be converted to a boolean as it is intended according to the method signature `function h($text, $double = true, $charset = null)`.

Reply:
I meant you should fix it first in `master`

Reply:
https://github.com/cakephp/cakephp/pull/12719